### ISSUE TITLE
Evernote (файл)

### ISSUE THREAD

**Issue by CB9TOIIIA** [zenden2k/image-uploader#111]
Возможно ли добавить в evernote в список загрузок? вполне удобно и практично
Ограничение до 100 МБ на 1 заметку.

Я ПА - в месяц до 4 ГБ можно грузить... вполне отлично: http://i.imgur.com/W8oQXIi.png

**Comment by vladios13** [comment]
Я за

**Comment by zenden2k** [comment]
"ПА" это что?

**Comment by CB9TOIIIA** [comment]
премиум аккаунт
Полезные материалы по API:
http://www.slideshare.net/evernote_ru/api-12756806
http://habrahabr.ru/company/evernote/blog/57807/
https://dev.evernote.com/doc/

**Comment by CB9TOIIIA** [comment]
Лимиты:
Текущие лимиты загрузок:

Бесплатный аккаунт: загружайте до 60 Мб данных в месяц, мы не ограничиваем общий объем ваших данных на сервере
Премиум-аккаунт: загружайте до 4 Гб данных каждый месяц, мы не ограничиваем общий объем ваших данных на сервере
Бизнес-аккаунт: Лимит загрузок для личных заметок каждого пользователя составляет 4 Гб в месяц. У каждой компании существует лимит для бизнес-заметок, распределенный между всеми пользователями. Этот лимит составляет 2 Гб для каждого места. Например, если в вашей компании 10 мест, ежемесячный лимит для бизнес-аккаунта составит 20 Гб. Мы не ограничиваем общий объем ваших данных на сервере.

https://evernote.com/intl/ru/contact/support/kb/#!/article/23283158

**Comment by zenden2k** [comment]
Для бесплатного аккаунта слишком маленький лимит, не вижу смысла, при наличии Яндекс Диска/Google Drive.

**Comment by CB9TOIIIA** [comment]
Спорно, т.к. я например уже в течение 5 лет пользуюсь Evernote, у меня там около 3000 заметок (статьи, файлы, др. данные) - есть расширение для evernote, которое при поиске в Яндексе или Гугле выводит смарт-бар (автоматический поиск еще в Ваших заметках) - это реально помогает, чем перебирать выдачу - есть своя небольшая БД.

Я согласен Evernote не предназначен для файлохранилища, но это инструмент, который помогает в учебе, работе, в жизни. Выбрал нес-ко word файлов и отправил в evernote + сделал заметки.

**Comment by CB9TOIIIA** [comment]
Также самой главной причиной выбрать evernote - это личный блокнот навсегда. Все файлопомойки удаляют данные и имеют срок хранения + не приватны.

**Comment by zenden2k** [comment]
Яндекс Диск имеет срок хранения? O_o

**Comment by arhangelsoft** [comment]
>Я согласен Evernote не предназначен для файлохранилища, но это инструмент, который >помогает в учебе, работе, в жизни. Выбрал нес-ко word файлов и отправил в evernote + сделал >заметки.

Image Uploader просто загружает файлы и картинки на хостинг. Заметки плодить вы не сможете.
А скриптами диалоги пока создавать нельзя.

**Comment by CB9TOIIIA** [comment]
в evernote есть функция создания заметки по email, т.е. отправить на почту специальную.
Есть ли возможность файлы отправлять на почту через image_uploader?
http://habrahabr.ru/company/evernote/blog/189906/

**Comment by arhangelsoft** [comment]
Идея кстати хорошая на счёт email клиента.

**Comment by zenden2k** [comment]
libcurl умеет IMAP и SMTP, но это бред и не нужно.

**Comment by CB9TOIIIA** [comment]
хм... почему логично же :)
Сделал скриншоты и сразу отправил на почту кому-то.
Не надо открывать браузер / сайт / прикреплять и отправлять.

**Comment by arhangelsoft** [comment]
Нет. не логично. Когда вы пишите письмо с картинками, вы пишите письмо, оформляете и т.д. и т.п. и прикрепляете файлы. Это нормальное письмо, его не засунут в спам сразу же.

Каким боком тут Image Uploader? 
Просто слать письма с вложениями без содержания и норм. заголовка? Это называется спам.

**Comment by CB9TOIIIA** [comment]
ну каждому свое, я просто всеми руками за Evernote
